### PR TITLE
ST: Add missing escape characters for log errors whitelist

### PR DESCRIPTION
**Pull request by Frawless**
### Type of change

- Bugfix

### Description

There were missing escape characters for two of the whitelisted errors that caused unexpected failures due to not-matching the whitelisted error. This PR should solve it.

### Checklist

- [ ] Make sure all tests pass

### CODE REVIEW

**Comment by Frawless** [comment]
TF failed due to https://github.com/strimzi/strimzi-kafka-operator/issues/9648 and one instance failed during provisioning. Azure is working fine and it doesn't fail due to CO logs error. This PR is now ready for review.

**Review by see-quick — APPROVED** [review]

**Review by im-konge — APPROVED** [review]

File: systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java [hunk 1]
```diff
@@ -40,6 +41,7 @@
  */
 @Tag(REGRESSION)
 @Tag(INTERNAL_CLIENTS_USED)
+@Tag(ARM64_UNSUPPORTED) // Due to https://github.com/strimzi/strimzi-kafka-operator/issues/9648
```
Comment (im-konge) [inline systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java:44]: The correct issue is https://github.com/strimzi/test-clients/issues/75

File: systemtest/src/test/java/io/strimzi/systemtest/operators/topic/TopicST.java [hunk 2]
```diff
@@ -140,6 +141,7 @@ void testCreateTopicViaKafka(ExtensionContext extensionContext) {
         verifyTopicViaKafka(Environment.TEST_SUITE_NAMESPACE, testStorage.getTopicName(), topicPartitions, KAFKA_CLUSTER_NAME);
     }
 
+    @Tag(ARM64_UNSUPPORTED) // Due to https://github.com/strimzi/strimzi-kafka-operator/issues/9648
```
Comment (im-konge) [inline systemtest/src/test/java/io/strimzi/systemtest/operators/topic/TopicST.java:144]: The correct issue is https://github.com/strimzi/test-clients/issues/75

**Review by im-konge — APPROVED** [review]

**Pull request merged**